### PR TITLE
fix(button): replace style tag with classname in stories

### PR DESCRIPTION
**Pull request by Rashi-Gupta-2000**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

File: core/components/atoms/button/__stories__/style.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.SplitButton-dropdown {
+    width: 150px;
```
Comment (satyamyadav) [inline core/components/atoms/button/__stories__/style.css]: Please use spacing token here.

File: core/components/atoms/button/__stories__/SplitButton.story.jsx [hunk 2]
```diff
@@ -19,7 +20,7 @@ export const splitButton = () => {
       <Button className="mr-2" aria-label="Request review">
         Request review
       </Button>
-      <div className="mb-10" style={{ width: '150px' }}>
+      <div className="mb-10 SplitButton-dropdown">
```
Comment (satyamyadav) [inline core/components/atoms/button/__stories__/SplitButton.story.jsx]: Please verify and remove this styling as there is no need for this width and removing it from the story does not affect anything.

**Review by satyamyadav — CHANGES_REQUESTED** [review]

File: core/components/atoms/button/__stories__/SplitButton.story.jsx (hunk 2 above)
Comment (Rashi-Gupta-2000) [reply core/components/atoms/button/__stories__/SplitButton.story.jsx]: It does affect the dropdown width which opens on click of the split button

Comment (satyamyadav) [reply core/components/atoms/button/__stories__/SplitButton.story.jsx]: It does not seem to be do so, I tried on this sandbox https://codesandbox.io/s/sleepy-violet-sgteih?file=/index.js

Comment (satyamyadav) [reply core/components/atoms/button/__stories__/SplitButton.story.jsx]: Also to control Dropdown width, it has props like `width`, `minWidth` and `maxWidth`.

**Review by satyamyadav — DISMISSED** [review]

**Review by satyamyadav — DISMISSED** [review]

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**